src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -62,7 +62,7 @@ export class InputActionDialog {
       } else if (!jenkinsNamespace) {
         console.log("Warning no jenkinsNamespace on the Build!")
       } else {
-        url = pathJoin(forgeUrl, "/services/jenkins/", jenkinsNamespace, url);
+        url = pathJoin(forgeUrl, "/api/openshift/services/jenkins/", jenkinsNamespace, url);
```

Comment:
the change looks fine but the commit syntax is wrong

Reply:
I think we can fix the commit syntax when squashing and merging this PR?

Reply:
that will work too

Reply:
How about `fix(#1771): Add a URL prefix to forge API`? Does it have to be `fix(launcher)`?

Reply:
the thing in parens is the subject, not the issue number. I could be wizard or build or planner. It sorts the changes by subject.